cypress/integration/Toggle.spec.js

```diff
@@ -0,0 +1,91 @@
+/// <reference types="Cypress" />
+
+/* eslint-disable no-undef */
+describe('The Toggle component', () => {
+  beforeEach(() => {
+    cy.visit(
+      '/iframe.html?selectedKind=Action%7CToggle&selectedStory=basic%20toggle&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel'
+    );
+
+    // Add data-id attribute
+    cy.get('label > input').invoke('attr', 'data-id', 'toggle-input');
+  });
+
+  it('Toggles!', () => {
+    cy.get('[data-id="toggle-input"]').should('not.be.checked');
+    cy.get('label').click();
+    cy.get('[data-id="toggle-input"]').should('be.checked');
+    cy.get('[data-id="toggle-input"]').uncheck({ force: true });
+    cy.get('[data-id="toggle-input"]').should('not.be.checked');
+    cy.get('[data-id="toggle-input"]').check({ force: true });
+    cy.get('[data-id="toggle-input"]').should('be.checked');
+    cy.get('[data-id="toggle-input"]').uncheck({ force: true });
+    cy.get('[data-id="toggle-input"]').should('not.be.checked');
+  });
+});
+
+/* eslint-disable no-undef */
+describe('Cannot Toggle component (disabled)', () => {
```

Comment:
In general, we try to group our `describe()` statements inside the initial one - this is something I only learned was possible when I started working here:

```js
describe('The toggle component', () => {
  describe('the disabled state', () => {
    // tests go here
  });
});
```

This also makes it a little more DRY so you don't have to repeat your `beforeEach()`!